Components/Core/Spells/Equivalencies.lua

```diff
@@ -196,30 +196,31 @@ TMW.BE = {
 		},
 		Slowed = {
 			   -116, -- Frostbolt                           (mage, frost)
-			   -120, -- Cone of Cold                        (mage, frost)
```

Comment:
Sorry, missed this one. Cone of Cold being checked by name is intentional for the same reasons as the others I mentioned. You can add the `-` to `212792` if you want, instead of `120`